yarn-project/aztec-nr/aztec/src/history/nullifier_inclusion.nr

```diff
@@ -5,7 +5,29 @@ use crate::{
     oracle::get_nullifier_membership_witness::get_nullifier_membership_witness,
 };
 
-pub fn prove_nullifier_inclusion(
+pub fn prove_nullifier_inclusion(nullifier: Field, context: PrivateContext) {
+    // 1) Get the membership witness of the nullifier
```

Comment:
We can do the same as above to avoid code duplication.

Reply:
When doing that, it might make sense to have a `note_is_nullified` where passing in the note and computing the nullifier is done.
```rust
let nullifier = compute_siloed_nullifier(note, context);
```

This would be an additional function, as we need to ability to just check directly for nullifiers for those cases where there is no actual note such as initialization checks.